frontend/packages/gitops-plugin/src/plugin.tsx

```diff
@@ -1,50 +0,0 @@
-import * as _ from 'lodash';
-import { ModelDefinition, RoutePage, Plugin, ModelFeatureFlag } from '@console/plugin-sdk';
-import { FLAG_OPENSHIFT_GITOPS } from './const';
-import * as models from './models';
-
-type ConsumedExtensions = ModelDefinition | ModelFeatureFlag | RoutePage;
-
-const plugin: Plugin<ConsumedExtensions> = [
-  {
-    type: 'ModelDefinition',
```

Comment:
Note this extension was removed without replacement. I could not issues that appeared after removal but this may require more testing